src/utils.js

```diff
@@ -103,3 +106,27 @@ export const codeshift = {
     })
   }
 };
+
+// gather list of migrations & make it easy to override for testing
+export const migration = {
+  async load() {
+    const MIGRATIONS = [
+      'capybara',
+      'cypress',
+      'ember',
+      'nightmare',
+      'nightwatch',
+      'protractor',
+      'puppeteer',
+      'selenium-java',
+      'selenium-javascript',
+      'selenium-python',
+      'testcafe',
+      'webdriverio'
+    ];
+
+    return Promise.all(MIGRATIONS.map(async m => {
+      return (await import(`./migrations/${m}.js`)).default;
+    }));
```

Comment:
Not a fan of consts not at the root level. Also since we're only using this variable exactly once, could inline it.

```suggestion
    return Promise.all([
      'capybara',
      'cypress',
      'ember',
      'nightmare',
      'nightwatch',
      'protractor',
      'puppeteer',
      'selenium-java',
      'selenium-javascript',
      'selenium-python',
      'testcafe',
      'webdriverio'
    ].map(async m => (
       await import(`./migrations/${m}.js`)
    ).default));
```